<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>设备参数</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>连续数据反馈</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_31">
           <property name="maximum">
            <number>99999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_31">
           <property name="text">
            <string> 设置</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>系统参数</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="2">
            <widget class="QCheckBox" name="checkBox_24">
             <property name="text">
              <string>报警需要连续发送波形</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>远端数据通讯格式</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>恢复出厂设置</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QComboBox" name="comboBox_6">
             <item>
              <property name="text">
               <string>485</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LORA私有协议+485</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WIFI私有协议+485</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NB+485</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WIFI MQTT+485</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WIFI MODBUS主机+485</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WIFI MODBUS从机+485</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>RS485波特率</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="7" column="2">
            <widget class="QComboBox" name="comboBox_9_3">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QPushButton" name="pushButton_6_9">
             <property name="text">
              <string>设置参数</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QComboBox" name="comboBox_9_1">
             <item>
              <property name="text">
               <string>无校验</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>奇校验</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>偶校验</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QComboBox" name="comboBox_9_2">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>校验位</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QSpinBox" name="spinBox_7">
             <property name="maximum">
              <number>115200</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>开启报警检测</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>数据配置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Y轴报警值</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>加速度</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox_14">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>速度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_15">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox_16">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>振幅</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string> Z轴报警值</string>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>加速度</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox_17">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>速度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_18">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox_19">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>振幅</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_11_20">
             <property name="text">
              <string>设置参数</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>其它</string>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>采样率</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox_3">
                <property name="maximum">
                 <number>999999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>采样间隔</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_21">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox_20">
                <property name="minimum">
                 <number>-9999</number>
                </property>
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>温度预警</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>X轴报警值</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>加速度</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox_11">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>速度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_12">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox_13">
                <property name="maximum">
                 <number>99999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>振幅</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>连续震动采集长度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_23">
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_23">
           <property name="text">
            <string>设置</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>连续数据时间间隔</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_22">
           <property name="maximum">
            <number>99999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_22">
           <property name="text">
            <string> 设置</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>信道</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_2">
           <property name="maximum">
            <number>99999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string> 设置</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>空中速率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_5">
           <property name="maximum">
            <number>99999999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string> 设置</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>设备数据</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0,0,1" rowminimumheight="0,0,0,0">
       <item row="0" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>连续数据类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>X轴加速度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>X轴速度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>X轴位移</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Y轴加速度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Y轴速度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Y轴位移</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Z轴加速度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Z轴速度</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Z轴位移</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_reqData">
           <property name="text">
            <string>请求连续数据</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_chart_0">
         <property name="text">
          <string>X轴加速度</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_chart_1">
         <property name="text">
          <string>X轴速度</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="checkBox_chart_2">
         <property name="text">
          <string>X轴位移</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="checkBox_chart_3">
         <property name="text">
          <string>Y轴加速度</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="checkBox_chart_4">
         <property name="text">
          <string>Y轴速度</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QCheckBox" name="checkBox_chart_5">
         <property name="text">
          <string>Y轴位移</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_chart_6">
         <property name="text">
          <string>Z轴加速度</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBox_chart_7">
         <property name="text">
          <string>Z轴速度</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="checkBox_chart_8">
         <property name="text">
          <string>Z轴位移</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0" colspan="6">
        <widget class="DeviceChartView" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButton_back">
     <property name="text">
      <string> &lt;&lt;返回</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_slave">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceChartView</class>
   <extends>QWidget</extends>
   <header location="global">devicechartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
